kbs/src/prometheus/mod.rs

```diff
@@ -57,6 +57,78 @@ lazy_static! {
         Histogram::with_opts(response_sizes_opts).unwrap()
     };
 
+    /// KBS Policy Evaluations Total
+    pub(crate) static ref RESOURCE_POLICY_EVALS: Counter = {
+        let opts = Opts::new(
+            "resource_policy_evaluations_total",
+            "Total count of resource policy evaluations",
+        );
+        Counter::with_opts(opts).unwrap()
+    };
+
+    /// KBS Policy Violations Total
+    pub(crate) static ref RESOURCE_POLICY_VIOLATIONS: Counter = {
+        let opts = Opts::new(
+            "resource_policy_violations_total",
+            "Total count of resource policy violations",
+        );
+        Counter::with_opts(opts).unwrap()
+    };
+
+    /// KBS Policy Errors Total
+    pub(crate) static ref RESOURCE_POLICY_ERRORS: Counter = {
+        let opts = Opts::new(
+            "resource_policy_errors_total",
+            "Total count of errors during policy evaluation",
+        );
+        Counter::with_opts(opts).unwrap()
+    };
+
+    /// KBS Attestation Requests Total
+    pub(crate) static ref ATTESTATION_REQUESTS: Counter = {
+        let opts = Opts::new(
+            "attestation_requests_total",
+            "Total count of attestation requests",
+        );
+        Counter::with_opts(opts).unwrap()
+    };
+
+    /// KBS Attestation Failures Total
+    pub(crate) static ref ATTESTATION_FAILURES: Counter = {
+        let opts = Opts::new(
+            "attestation_failures_total",
+            "Total count of attestation failures",
+        );
+        Counter::with_opts(opts).unwrap()
+    };
+
+    /// KBS Attestation Errors Total
+    pub(crate) static ref ATTESTATION_ERRORS: Counter = {
+        let opts = Opts::new(
+            "attestation_errors_total",
+            "Total count of errors during attestation processing",
+        );
+        Counter::with_opts(opts).unwrap()
+    };
+
+    /// KBS Auth Requests Total
+    pub(crate) static ref AUTH_REQUESTS: Counter = {
+        let opts = Opts::new(
+            "auth_requests_total",
+            "Total count of auth requests",
+        );
+        Counter::with_opts(opts).unwrap()
+    };
+
+    /// KBS Auth Errors Total
+    pub(crate) static ref AUTH_ERRORS: Counter = {
+        let opts = Opts::new(
+            "auth_errors_total",
+            "Total count of errors during auth processing",
+        );
+        Counter::with_opts(opts).unwrap()
+    };
```

Comment:
Oh, the same pattern of code breaks out here. It seems that we can consider using macros to simplify it. We can add it in the subsequent PR